src/main/scala/org/constellation/domain/transaction/TransactionService.scala

```diff
@@ -37,7 +37,7 @@ class TransactionService[F[_]: Concurrent](val transactionChainService: Transact
   def createDummyTransactions(count: Int): F[List[TransactionCacheData]] =
     List
       .fill(count) {
-        val keyPair = KeyUtils.makeKeyPair()
+        val keyPair = KeyUtils.makeKeyPair()//todo note creation here is expensive, we should just use the node keypair
```

Comment:
we cannot use node keypair for dummy transaction, since both src and dst must be different than the node address to not interfere with rate limiting, tx chain, and so on